input/cql/DementiaCognitiveAssessmentFHIR.cql

```diff
@@ -52,6 +59,9 @@ define "Assessment of Cognition Using Standardized Tools or Alternate Methods":
     where CognitiveAssessment.value is not null
       and CognitiveAssessment.status in { 'final', 'amended', 'corrected', 'preliminary' }
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
This should be the union of ConditionEncounterDiagnosis and ConditionProblemsHealthConcerns